### ISSUE TITLE
New line in functions. Not good.

### ISSUE THREAD

**Issue by NikKovIos** [raywenderlich/swift-style-guide#341]
I want to talk about this
> ![image](https://user-images.githubusercontent.com/16647690/109800585-6af96080-7c2e-11eb-86a4-cfaea5799926.png)
> ![image](https://user-images.githubusercontent.com/16647690/109800596-7056ab00-7c2e-11eb-95c5-122258c6b870.png)

Working for a while with swift and xcode and find out, that it's not convenience when someone wrap a string to a new line. And I upset, that my favorite guide tells to do it.

----

**Advantages**:  
- less line width

**Disadvantages**:  
- hard to find a method. For example you have methods `func test(one:...)` and `func test(two:...)`. In that case it's hard to find the needed method. In search field you will type `test(`. If not wrap you can search via `test(one` or `test(two`.
- indentation is more visually better if not wrapping:

```
    func test(one: String,
              two: String,
              three: String) {

    }

    func test(
        one: String,
        two: String,
        three: String
    ) {

    }
```

Summing up, I suggest to change the guide.

**Comment by hollance** [comment]
While style is often a matter of personal taste, the reason for this wrapping is probably that it prevents code blocks from becoming too wide on the website and in the books.

**Comment by rcritz** [comment]
As Matthijs said, our style guide is predicated on our needs for publishing and that's why we wrap the way we do.

I'll note that if you use the Xcode jump bar to search for your functions (either clicking in it or using the Control-6 shortcut), the wrapping has no impact on your searching.
